src/main/java/fi/metatavu/kuntaapi/server/integrations/ptv/updaters/PtvServiceChannelEntityDiscoverJob.java

```diff
@@ -485,7 +485,7 @@ private void updateParentPageIds(List<PageId> parentPageIds) {
       if (ManagementConsts.IDENTIFIER_NAME.equals(parentPageIdentifier.getSource())) {
         // TODO: REFACTOR THIS AWAY FROM THE PTV INTEGRATION PLUGIN
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)